runtime/onert/backend/train/ops/ElementwiseActivationLayer.h

```diff
@@ -0,0 +1,63 @@
+/*
+ * Copyright (c) 2023 Samsung Electronics Co., Ltd. All Rights Reserved
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *      http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+#ifndef __ONERT_BACKEND_TRAIN_OPS_ELEMENTWISEACTIVATIONLAYER_H__
+#define __ONERT_BACKEND_TRAIN_OPS_ELEMENTWISEACTIVATIONLAYER_H__
+
+#include <backend/IPortableTensor.h>
+#include <ops/ElementwiseActivationLayer.h>
+
+#include <exec/train/ITrainableFunction.h>
+
+namespace onert
+{
+namespace backend
+{
+namespace train
+{
+namespace ops
+{
+
+enum class ElementwiseActivationType
+{
+  kReLU,
+};
+
+class ElementwiseActivationLayer : public ::onert::exec::train::ITrainableFunction,
+                                   public cpu::ops::ElementwiseActivationLayer
+{
+public:
+  ElementwiseActivationLayer();
+
+  void configure(const IPortableTensor *input, IPortableTensor *output,
+                 const IPortableTensor *grad_input, IPortableTensor *grad_output, float alpha,
+                 float beta, ElementwiseActivationType op_type);
+  void forward(bool training) override;
+  void backward() override;
+
+private:
+  const IPortableTensor *_grad_input;
+  IPortableTensor *_grad_output;
```

Comment:
```suggestion
  IPortableTensor *_grad_input;
  const IPortableTensor *_grad_output;
```

If `_grad_input` is on the same location as `_input` of `cpu::ops::ElementwiseActivationLayer` in the graph, `_grad_input` will be updated instead of `_grad_output` in this layer.

Reply:
In forward pass, the output tensor will be updated. Am I right? Then I think both tensors should be non-const tensor.